src/components/manage-hospitals/index.tsx

```diff
@@ -96,8 +99,9 @@ export const ManageHospitals: FunctionComponent<ManageHospitalsProps> = (props)
         case "edit-supervisors":
           await patchHospital(selectedHospital.id, { supervisor: items.length ? items[0].id : null });
       }
+      Notify.success(i10n("defaultSuccessMessage"));
```

Comment:
As a note. `i10n` is the wrong term. Only this exists
`i18n` stads for `Internationalization` (the process of adding multiple languages to your app)
`l10n` stads for `Localization` (first letter is L) (the process of translating your app)

Reply:
Yeah, @iamandrewluca, you're absolutely right. Changed it over the project. Thanks